src/parsers/uniqueTokens.js

```diff
@@ -91,9 +91,8 @@ export const parseAccountUniqueTokens = data => {
             : collection.name,
         id: token_id,
         isSendable:
-          asset_contract.nft_version === '1.0' ||
-          asset_contract.nft_version === '3.0' ||
-          asset_contract.schema_name === 'ERC1155',
+          asset_contract.contract_standard === 'ERC721' ||
+          asset_contract.contract_standard === 'ERC1155',
```

Comment:
```suggestion
          SUPPORTED_SENDABLE_NFT_CONTRACTS.includes(asset_contract.contract_standard)
```

```
const SUPPORTED_SENDABLE_NFT_CONTRACTS = ['ERC721', 'ERC1155'];
```

Reply:
got the v1 & v2 endpoints mixed up, since we're gonna be checking 2 diff properties im gonna not do this